_posts/2016/03/2016-03-08-proposal-instructor-trainees.md

```diff
@@ -0,0 +1,83 @@
+it's unlikely they will have much time to commit. By doing that kind of questioning, 
+I was able to eliminate quite a few people from the people who originally volunteered to come.  
+
+The 20 people I ended up with were what I considered the best 20. 
+To get a place, they had to complete the prerequisite activities by a deadline, 
+or lose the spot - which is one good way to tell whether or not people are serious. 
+If applicants aren't prepared to complete tasks, or if they're sloppy about deadlines, 
+chances are they won't teach workshops either.  
+
+My 20 all did the prep, they turned up on the day, they stuck it out with no falling by the wayside.
+
+After the workshop, we were lucky we had the big Research Bazaar (ResBaz) event 
+coming up at which we offered parallel Software Carpentry workshops in both R and Python. 
+Several fledgling instructors got a chance to teach there while being assisted (and observed) 
+by more experienced instructors. After ResBaz, I organised a Hangout practice session for 
+seven of those recent instructor trainees. During that session, they all taught to the group 
+and they all critiqued one another face to face and via an etherpad. I have since done a second practice teaching 
+session with two other trainees (and walked them both through the pull request requirement) 
+and am planning a third with more trainees from Townsville. I have also followed up twice 
+via email with the rest who attended from Sydney and Canberra.
+
+I think my follow ups have helped people complete, and my ability to get a lot of trainees 
+teaching soon after the training ended focused them on the importance of getting through the final stages. 
+I think having a local 'shepherd' can really help get people over the line. 
+But the shepherd could be anywhere - it's more just having someone specific to whom trainees feel 
+*accountable* - someone who follows up with them, chases them up, and actually cares that 
+they get the final bits done and qualify as instructors.
+
+Maybe if there is no-one local who could play that role,  this could be a job for the mentoring committee to handle - 
+it really is a mentoring task. 
+
+The ideal would be to have the instructor trainer 
+hand the class over to the mentor as part of the final session of the training,
+and for the mentor to check in with trainees regularly from then on – 
+maybe by running teaching practice sessions, handholding, or talking them 
+through the final tasks, getting them over their nerves about actually 
+getting up and teaching (the practice sessions help with that). 
+
+I think there also needs to be some contribution pathway 
+for people who've completed instructor training but have discovered - belatedly! - 
+that teaching is not for them. In order not to waste their training, 
+they could become an organiser/cheerleader. We could always use more of those.
+
+-- Belinda Weaver, Brisbane, Australia
```

Comment:
Your name will automatically show up on the post (because of the 'authors' field at the top), so this line won't be needed.